tensorboard/webapp/runs/data_source/BUILD

```diff
@@ -10,11 +10,25 @@ ng_module(
     srcs = [
         "runs_data_source.ts",
         "runs_data_source_module.ts",
+    ],
+    data = [
+        "//tensorboard/components/tf_backend",
```

Comment:
The `TBRunsDataSource` impl in `runs_data_source.ts` assumes tfBackend is defined and available.  The build & tests pass even without this data dep, but I thought it would be better to explicitly state its dependency in some way.

Reply:
Hmm, I think if done poorly, the tf_backend code will end up with the webfiles.zip which is not what we want. Even if there is an implicit dependency, don't create the data dep unless required.

Reply:
Done!